ui/ui-components/pages/model/[modelId]/source/[sourceId]/multipleProperties.tsx

```diff
@@ -79,18 +77,15 @@ export default function Page(props) {
       if (property.filters?.length > 0) continue;
 
       let optMatch = true;
-      for (const key of Object.keys(property.options)) {
+      for (const [key, value] of Object.entries(property.options)) {
```

Comment:
This change was subtle, but `key` may have been `undefined`, which was making the later lookup crash silently (thanks react :( )